src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -516,7 +516,7 @@ export class QuickOpenWidget implements IModelProvider {
 
 		// Trigger open of element on selection
 		if (this.isVisible()) {
-			let mode = preferredMode || Mode.OPEN;
+			let mode = (event && event.payload && event.payload.originalEvent && event.payload.originalEvent.middleButton ? Mode.OPEN_IN_BACKGROUND : false) || preferredMode || Mode.OPEN;
```

Comment:
I would prefer if this was done at the place where the event happens ([here](https://github.com/Microsoft/vscode/blob/master/src/vs/base/parts/quickopen/browser/quickOpenWidget.ts#L256)) and then passed over as `preferredMode`